File: 100joon/Sliver/11399.py
import sys

n = int(input())
withdraw_time = list(map(int, sys.stdin.readline().split()))
withdraw_time.sort()
spend_time = []
t = 0

for i in withdraw_time:
    t += i
    spend_time.append(t)


print(sum(spend_time))

File: 100joon/Sliver/11399.js
const input = require("fs")
  .readFileSync("input.txt", "utf-8")
  .trim()
  .split("\n");

const nums = input[1].split(" ").map(Number);
nums.sort((a, b) => a - b);

let sum = 0,
  total = 0;

for (const num of nums) {
  sum += num;
  total += sum;
}

console.log(total);
